xml_converter/generators/code_generator.py

```diff
@@ -267,6 +267,35 @@ def load_input_doc(self, dir_path: str) -> None:
                 content=document.content
             )
     
+    def write_parseable(self, output_directory: str) -> None:
```

Comment:
Lets add a `print()` here to match the `print()` found at the beginning of `write_webdocs()`